modules/tensor_mechanics/src/materials/ComputeStressBase.C

```diff
@@ -44,46 +44,70 @@ ComputeStressBase::ComputeStressBase(const InputParameters & parameters)
     _elasticity_tensor(getMaterialPropertyByName<RankFourTensor>(_elasticity_tensor_name)),
     _extra_stress(getDefaultMaterialProperty<RankTwoTensor>(_base_name + "extra_stress")),
     _Jacobian_mult(declareProperty<RankFourTensor>(_base_name + "Jacobian_mult")),
-    _store_stress_old(getParam<bool>("store_stress_old"))
+    _store_stress_old(getParam<bool>("store_stress_old")),
+    // InitialStress TODO: remove the following
+    _initial_stress_provided(getParam<std::vector<FunctionName>>("initial_stress").size() ==
```

Comment:
Mark the initial stress parameter above as deprecated?

Reply:
Yep, i'm going to mark the initial stress parameter as deprecated, but can't do that until this PR gets merged, then i can change Mastodon, and then change to the deprecatedParam